README.md

```diff
@@ -103,6 +105,8 @@ This tools contains many subcommands to help you doing what you need.
 - [start](#start): starts your app previously built with the build command
 - [logs](#logs): displays output from the containers
 - [stop](#stop): stops your app previously started with the start command
+- [dev](#dev): starts your app in dev mode
+- [init](#init): Generates Docker and Docker Compose files
```

Comment:
```suggestion
- [init](#init): generates Dockerfile and Docker Compose files
```

Reply:
It not just the Dockerfile, but also the `.dockerignoer`